app/models/conference_room.rb

```diff
@@ -0,0 +1,11 @@
+class ConferenceRoom < ApplicationRecord
+  before_validation { self.color&.downcase! }
+
+  validates :capacity, presence: true
+  #validates that color format is like: #FF0ddE
```

Comment:
A comment saying what a piece of code does usually means there's something wrong with the code ;) You could extract the regexp to a constant named `HEX_COLOR_PATTERN` or something similar